source/adminguide/management.rst

```diff
@@ -587,3 +587,52 @@ To start the Management Server:
 
    # service cloudstack-management start
 
+
+Management Server Statistics and Peers
+--------------------------------------
+
+Administrators are able to view the statistics and peers information of management server.
+      
+#. Log in to the CloudStack UI as administrator
+
+#. In the left navigation bar, click Infrastructure.
+
+#. Click "Management servers", all management servers are listed.
+
+|management-servers-list.png|
+
+#. Click the management server you'd like to view. The statistics of the management server are displayed.
+
+|management-server-statistics.png|
+
+#. Navigate to the "Peers" tab. The peers of the management servers are listed
+
+|management-server-peers.png|
+
```

Comment:
@weizhouapache can rename the ms and peer names, and update the screenshots?

Reply:
will do

Reply:
updated